nix/cardano-services/deployments/default.nix

```diff
@@ -206,6 +212,19 @@ in
                   port.name = "http";
                 };
               }
+              {
+                pathType = "Prefix";
+                path = let
+                  inherit (oci.meta) versions;
+                 in
+                  lib.concatLists [
+                    (map (v: "/v${v}/chain-history") versions.chainHistory)
+                  ];
+                backend.service = {
+                  name = "${final.namespace}-chain-history-provider";
+                  port.name = "http";
+                };
+              }
```

Comment:
```suggestion
    
```

im pretty sure this is not necessary since we have a patch in ingress

Reply:
confirmed, after initial apply, removed this and re-apply shows no diff, removing